src/app/components/views/homepage/homepage.component.ts

```diff
@@ -48,7 +51,7 @@ export class HomepageComponent implements OnInit, OnDestroy {
   }
 
   /**
-   * @description For use in html when the refresh button is clicked. Will call JobService 
+   * @description For use in html when the refresh button is clicked. Will call JobService
```

Comment:
@looksliketyler is it a big deal that three lines here had a trailing whitespace removed?